config/swagger.yaml

```diff
@@ -0,0 +1,140 @@
+          description: Webhook routed correctly
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+    get:
+      summary: Gets the configuration for the given webhook
+      operationId: get_route
+      responses:
+        '200':
+          description: The correct webhook
+          schema:
+            $ref: "#/definitions/Route"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+    patch:
+      summary: Edit the configuration of the given webhook
+      operationId: patch_route
+      parameters:
+        - name: new_info
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/PatchRoute"
+      responses:
+        '204':
+          description: Routing patched correctly
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+  /routes/{token}/regenerate:
+    parameters:
+      - name: token
+        in: path
+        required: true
+        description: The token of the route
+        type: string
+    post:
+      summary: Regenerates the token for the given route
+      operationId: regenerate_token
+      responses:
+        '200':
+          description: Successful regeneration, returns json data of the new route
+          schema:
+            $ref: "#/definitions/Route"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+  /routes:
+    get:
+      summary: Gets all routes that are configured for given user
+      operationId: get_all_routes
+      responses:
+        '200':
+          description: Returns the given routes
+          schema:
+            $ref: "#/definitions/Routes"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+  /add-route:
+    post:
+      summary: Gets all routes that are configured
+      operationId: add_route
+      parameters:
+        - name: new_route
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/NewRoute"
+      responses:
+        '201':
+          description: Expected response to a valid request
+          schema:
+            $ref: "#/definitions/Route"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+definitions:
+  NewRoute:
+    required:
+      - name
+      - destination
+    properties:
+      name:
+        type: string
+      destination:
+        type: string
+  PatchRoute:
+    properties:
+      name:
+        type: string
+      destination:
+        type: string
+  Route:
+    allOf:
+      - $ref: "#/definitions/NewRoute"
+      - required:
+          - token
+        properties:
+          token:
+            type: string
+          owner:
+            type: string
+  Routes:
```

Comment:
It's generally a bad idea to have plural class names - try RouteList or similar.